src/main/webapp/pages/analysis.html

```diff
@@ -5,20 +5,13 @@
   data-layout-decorate="~{template/page}"
 >
   <head>
-    <title th:text="#{analysis.title(${analysisSubmission.getName()})}">ANALYSIS RESULTS</title>
+    <title th:text="#{analysis.title(${analysisName})}">ANALYSIS RESULTS</title>
     <link rel="stylesheet" th:href="@{/dist/css/analysis.bundle.css}" />
 
     <script th:inline="javascript">
+
       window.PAGE = {
-        analysis: [[${analysisSubmission}]],
-        analysisName: [[${analysisSubmission.getName()}]],
-        analysisState: [[${analysisSubmission.getAnalysisState()}]],
         analysisType: [[${analysisType}]],
```

Comment:
Why are you keeping this one variable on the page?

Reply:
Removed in 8d63f91593b46a445bb918bea869f802e346b15f